woocommerce/woocommerce
# Hide the customizer when the active theme is a FSE theme

### All Submissions:






### Changes proposed in this Pull Request:
This PR makes sure that when a block theme is active, activating WooCommerce core will not enable the `Customizer`  option from the Appearance settings section and the `Customize` button from the WP admin

For more context see: https://woofse.wordpress.com/2022/06/07/pt-ensure-customizer-doesnt-show-up-with-block-themes-active-2/

### How to test the changes in this Pull Request:

1. Enable `WooCommerce`.
2. Enable the `Storefront` theme.
3. Check that the `Customize` button appears on the `Appearance` menu and on the menu bar.
4. Enable a block theme (i.e. `Twentytwentytwo`).
5. Check that the `Customize` button does not show up on the `Appearance` menu or on the menu bar.

### Other information:




### FOR PR REVIEWER ONLY:
